netutils/config/parser.py

```diff
@@ -156,7 +157,7 @@ def get_leading_space_count(config_line):
         """
         return len(config_line) - len(config_line.lstrip())
 
-    def _remove_parents(self, line, current_spaces):
+    def _remove_parents(self, line: str, current_spaces: int) -> t.Tuple:  # type: ignore
```

Comment:
`t.Tuple[str, ...]` as the return type should be able to get rid of the `# type: ignore`. Correspons to a tuple of varying size with `str` elements.